### PR TITLE
Update `.npmignore` and `.gitignore` files

### PR DESCRIPTION
**Pull request by Turbo87**
Resolves https://github.com/mike-north/ember-api-actions/issues/410

```
npm notice === Tarball Details ===
npm notice name:          ember-api-actions
npm notice package size:  8.3 kB
npm notice unpacked size: 26.2 kB
```

/cc @mike-north

### CODE REVIEW

**Comment by Turbo87** [comment]
@mike-north not sure what's going on, but the failures seem unrelated to the contents of this PR 🤔

**Comment by Turbo87** [comment]
@mike-north any updates on this? do you need help maintaining this addon?

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,7 @@
 # compiled output
 /dist/
 /tmp/
+*.tgz
```
Comment (alexlafroscia) [inline .gitignore:6]: @Turbo87 what are these `tgz` files?

Comment (alexlafroscia) [reply .gitignore:6]: I looked through the `unpkg` link and didn't see them anywhere in the actual published package...

Comment (Turbo87) [reply .gitignore:6]: the output of `npm pack`. I often run `npm pack` before `npm publish` to see what files are added to the published artifact.

Comment (Turbo87) [reply .gitignore:6]: to avoid issues like https://github.com/mike-north/ember-api-actions/issues/410 😉

Comment (alexlafroscia) [reply .gitignore:6]: Gotcha -- yeah, this configuration change looks right to me!

**Pull request merged**